packages/newspringchurchapp/src/user-settings/index.js

```diff
@@ -88,7 +88,11 @@ class UserSettings extends PureComponent {
                       <TableView>
                         <Touchable
                           onPress={() =>
-                            openUrl('https://newspring.cc/privacy')
```

Comment:
this means changing the `WebBrowser` did infact introduce breaking changes, that should be fixed in core too.

Reply:
Well, we just never changed these links to include the necessary arguments. What do we need to fix in core? Default to empty strings if the argument isn't there or something?

Reply:
Yes, it should be able to handle blank second and third arguments